api/modules/budgeting.js

```diff
@@ -31,7 +31,9 @@ const budgeting = module.exports = (() => {
         if (client) {
             const dateIncurredOverride = stripeInvoice.metadata?.date_incurred
                 ? moment(stripeInvoice.metadata.date_incurred, 'YYYY-MM-DD')
-                : moment(stripeInvoice.created, 'X')
+                : stripeInvoice.status_transitions?.finalized_at
+                    ? moment.unix(stripeInvoice.status_transitions.finalized_at, 'x').utc()
```

Comment:
Change to simply `moment(stripeInvoice.status_transitions.finalized_at, 'X')` since it should just use the same formatting as `moment(stripeInvoice.created, 'X')` which was working fine before. Even if it seems to work there's no need to introduce different functions for this since Stripe will give the exact same format for both `finalized_at` and `created`